tests/test_output.py

```diff
@@ -52,6 +58,11 @@ def setup():
         if not exists(new_dir):
             os.makedirs(new_dir)
 
+    # create jsonlines feature files if they don't already exist
+    feature_files = [join(_my_dir, 'train', 'f{}.jsonlines'.format(i)) for i in range(5)]
+    if not all([exists(ff) for ff in feature_files]):
+        create_jsonlines_feature_files()
+
```

Comment:
This code block is copied a number of times in a few different files (granted it is only 3 lines, but still that suggests to me it could be more modular). Would it make sense to put more of this code into the 'create_jsonlines_feature_files()' function? Or maybe just into another util function that checks if the files exist before calling 'create_jsonlines_feature_files()'. Then we could replace these three lines with a single function call eg "create_feature_files_if_none_exist()" or something similar.

Reply:
Hmm, yeah may be `create_jsonlines_feature_files()` could check whether the files already exist and only then create them? I'll change that.